ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldesfragment/repository/LdesFragmentRepository.java

```diff
@@ -21,7 +21,7 @@ Optional<LdesFragment> retrieveMutableFragment(String viewName,
 
 	Optional<LdesFragment> retrieveRootFragment(String viewName);
 
-	Stream<LdesFragment> retrieveImmutableFragmentsOfView(String viewName);
+	Stream<LdesFragment> retrieveNonDeletedImmutableFragmentsOfView(String viewName);
```

Comment:
Got to say I find this rename more confusing.. I'd keep it at retrieveImmutableFragmentsOfView